src/components/CourseTable/index.jsx

```diff
@@ -3,22 +3,76 @@ import PropTypes from 'prop-types';
 import Helmet from 'react-helmet';
 import React from 'react';
 import { Link } from 'react-router-dom';
+import Select from 'react-select';
+import qs from 'query-string';
 
 import { Hyperlink, SearchField } from '@edx/paragon';
 import TableContainer from '../../containers/TableContainer';
-import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
 import ButtonToolbar from '../ButtonToolbar';
 import PageContainer from '../PageContainer';
 import StatusAlert from '../StatusAlert';
 import { getPageOptionsFromUrl, updateUrl } from '../../utils';
+import Pill from "../Pill";
+import { UNPUBLISHED, PUBLISHED} from "../../data/constants";
+import './CourseTable.scss';
 
 const orgWhitelist = process.env.ORG_WHITELIST ? process.env.ORG_WHITELIST.split(',') : [];
+const dot = (color) => ({
+  alignItems: 'center',
+  display: 'flex',
+
+  ':before': {
+    backgroundColor: color,
+    borderRadius: 10,
+    content: '" "',
+    display: 'block',
+    marginRight: 8,
+    height: 10,
+    width: 10,
+  },
+});
 
 class CourseTable extends React.Component {
+  state = {
+    filterGroups: [
+      {
+        label: 'Course Run Statuses',
+        options: [
+          { value: UNPUBLISHED, label: 'Unsubmitted', color: '#E2C018' },
+          { value: 'in_review', label: 'In review', color: '#e7e7e7' },
```

Comment:
Maybe a comment about this special value? Might maybe be worth moving to the constants.js file too, with that comment?